nixos/modules/services/networking/ntpd.nix

```diff
@@ -88,6 +91,21 @@ in
         default = [];
       };
 
+      extraConfig = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Extra configuration directives that should be added to
+          <literal>ntp.conf</literal>
+        '';
+      };
+
+      ntpPackage  = mkOption  {
```

Comment:
I would simply call this option just `package` as many other modules do (ntp is then also included in the toplevel attribute: `config.services.ntp.package`).

Reply:
good idea, fixed.